File: app/src/main/java/dev/ragnarok/fenrir/media/exo/ExoUtil.java
package dev.ragnarok.fenrir.media.exo;

import com.google.android.exoplayer2.ExoPlayer;

public class ExoUtil {
    public static void pausePlayer(ExoPlayer player) {
        player.setPlayWhenReady(false);
        player.getPlaybackState();
    }

    public static void startPlayer(ExoPlayer player) {
        player.setPlayWhenReady(true);
        player.getPlaybackState();
    }
}

File: app_filegallery/src/main/kotlin/dev/ragnarok/filegallery/media/exo/ExoUtil.kt
package dev.ragnarok.filegallery.media.exo

import androidx.media3.exoplayer.ExoPlayer

object ExoUtil {
    fun pausePlayer(player: ExoPlayer?) {
        player?.playWhenReady = false
        player?.playbackState
    }

    fun startPlayer(player: ExoPlayer?) {
        player?.playWhenReady = true
        player?.playbackState
    }
}